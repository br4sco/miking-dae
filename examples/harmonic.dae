variables
x, vx : Float
init
x = 1.
equations
x' = vx;
vx' + vx + x = 0.
output
{x, vx, vx'}
