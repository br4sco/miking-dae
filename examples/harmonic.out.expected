-0.00217026750348,-0.00538701356945,0.00755727961649
